File: BackEnd/NodeEditor.RestAPI/NodeEditor.Entities/NodeGroup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NodeEditor.Entities
{
    public class NodeGroup:EntityBase
    {
        public ICollection<Node> Nodes { get; set; }

        public User User { get; set; }
        public Guid UserId { get; set; }
    }
}

File: FrontEnd/node-editor/src/Models/NodeGroup.ts
import { Node } from "./Node";
import { User } from "./User";
import { EntityBase } from "./EntityBase";
import { NodeMap } from "flume";

export interface NodeGroup extends EntityBase {
    nodes: Node[];
    user?: User;
    userId?: string;
    name?:string;
    flumeNodeMap:string;
}